scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/data/TaskDataspaces.java

```diff
@@ -57,7 +59,7 @@
 
     String getGlobalURI();
 
-    void copyInputDataToScratch(List<InputSelector> inputFiles) throws FileSystemException;
+    void copyInputDataToScratch(List<InputSelector> inputFiles) throws FileSystemException, InterruptedException;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: org.objectweb.proactive.extensions.dataspaces.exceptions.FileSystemException, java.lang.InterruptedException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1160)